Resources/doc/cookbook/recipe_sortable_listing.rst

```diff
@@ -66,7 +66,7 @@ feature in your config.yml such as
 	stof_doctrine_extensions:
 	    orm:
 	        default:
-                sortable: true
+                    sortable: true
```

Comment:
I think it is indented with 5 spaces now :P See the red bars? These are tabs. Can you change them into 4 spaces?